api/src/main/java/com/github/retrooper/packetevents/wrapper/play/server/WrapperPlayServerTeams.java

```diff
@@ -186,31 +172,20 @@ public void read() {
         players = new ArrayList<>();
         if (teamMode == TeamMode.CREATE || teamMode == TeamMode.ADD_ENTITIES || teamMode == TeamMode.REMOVE_ENTITIES) {
             int size;
-            if (serverVersion == ServerVersion.V_1_7_10)
-                size = readShort();
-            else
-                size = readVarInt();
+            if (serverVersion == ServerVersion.V_1_7_10) size = readShort();
```

Comment:
adding braces "{" "}" is probably preferable as it becomes very unclear to read if there are no braces IMO and bugs can easily be introduced